### PR TITLE
Fix Typo and grammatical errors

### PR DESCRIPTION
**Pull request by dhruvx19**
**This pull request makes the following changes :**
 * Fixes issue  #138

** Original:
"tfaction is a framework for Monorepo to build high level Terraform Workflows by GitHub Actions."
Corrected:
"tfaction is a framework for a Monorepo to build high-level Terraform workflows using GitHub Actions."

** Original:
"Notify the result of CI to pull requests with tfcmt, github-comment, and reviewdog"
Corrected:
"Notify the results of CI to pull requests using tfcmt, github-comment, and reviewdog."

** Original:
"A commit is pushed automatically, so you don't have to update .terraform.lock.hcl yourself"
Corrected:
"A commit is pushed automatically, so you don't have to update .terraform.lock.hcl manually."

** Original:
"A commit is pushed automatically, so you don't have to format Terraform configuration yourself."
Corrected:
"A commit is pushed automatically, so you don't have to format Terraform configuration manually."

** Original:
"CI fails if there are changes, which enables to merge pull requests without changes safely."
Corrected:
"CI fails if there are changes, which enables you to merge pull requests without unexpected changes safely."

### CODE REVIEW

**Comment by suzuki-shunsuke** [comment]
Thank you for your contribution!

File: docs/overview.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ slug: /
...
+tfaction is a framework for a Monorepo to build high-level Terraform workflows using GitHub Actions.
 You don't have to run `terraform apply` in your laptop, and don't have to reinvent the wheel for Terraform Workflows anymore.
 
 ## Features
 
 - Run `terraform plan` in pull requests, and run `terraform apply` by merging pull requests into the default branch
 - [Dynamic build matrix for Monorepo](/tfaction/docs/feature/build-matrix)
   - CI is run on only changed working directories
-- Notify the result of CI to pull requests with tfcmt, github-comment, and reviewdog
+- Notify the results of CI to pull requests using tfcmt, github-comment, and reviewdog.
```
Comment (suzuki-shunsuke) [inline docs/overview.md:16]:
```suggestion
- Notify the results of CI to pull requests using tfcmt, github-comment, and reviewdog
```

I don't think a period is necessary at the end of lists. We don't add periods to other list elements.

Comment (suzuki-shunsuke) [inline docs/overview.md:16]: Just curious. I'm not so good at English.
In this context, is `using` more natural than `with`?

**Comment by suzuki-shunsuke** [comment]
@akssshh said that he worked on the same issue, so if you have more updates it's good to communicate with him on the issue.
- https://github.com/suzuki-shunsuke/tfaction-docs/issues/138#issuecomment-1742022468

File: docs/overview.md (hunk 1 above)
Comment (dhruvx19) [reply docs/overview.md]: "Using" in this context emphasizes that you are employing or utilizing the tools (tfcmt, github-comment, and reviewdog) to achieve the task of notifying CI results. It conveys a sense of active utilization.
"With" is a more general preposition. It's less specific about how you are using them.
Both "with" and "using" work in this context, and the choice between them depends on your preference and the specific style you want to convey.

Comment (suzuki-shunsuke) [reply docs/overview.md]: Thank you for your explanation!
I understood. As you suggested, I prefer `using` to `with` in this context.

**Review by suzuki-shunsuke — APPROVED** [review]

**Pull request merged**